src/components/constants/date.js

```diff
@@ -1,5 +1,5 @@
 export const THOUSAND_MILLI_SECONDS = 1000;
```

Comment:
do we really need this? isn't it a predefined quantity? 
instead of such big constant name it will be easy to write 1000

Reply:
It isn't a part of my PR.

Reply:
I guess an AO could make little changes and put it up here.

Reply:
But I feel ritik, it is telling us about an entity in the formula we are applying. So, it is good to put in a constant.

Reply:
If this is being used in multiple places and provides info to the devs on the importance of it we can keep it.
Ideally we don't need Thousand Milli seconds or fourteen days as it doesn't add much value.

Reply:
3,4,5 makes much sense and it is needed as well